Service-Broker-Setup.adoc

```diff
@@ -51,7 +51,7 @@ env:
     -----END CERTIFICATE-----
 
   # UAA_CA_CERT can be obtained with the command line:
-  #   kubectl get secret -n $NAMESPACE secret-$REVISION -o jsonpath='{.data.uaa-ca-cert}' | base64 -d
```

Comment:
For clarity, it might be better to change `NAMESPACE` to `CF_NAMESPACE` in lines 46-47, and here and the following line to `UAA_NAMESPACE`, so users don't use the wrong cert